distributed/client.py

```diff
@@ -1451,13 +1447,7 @@ def wait_for_workers(
             Time in seconds after which to raise a
             ``dask.distributed.TimeoutError``
         """
-        if n_workers is no_default:
-            warnings.warn(
-                "Please specify the `n_workers` argument when using `Client.wait_for_workers`. Not specifying `n_workers` will no longer be supported in future versions.",
-                FutureWarning,
-            )
-            n_workers = 0
-        elif not isinstance(n_workers, int) or n_workers < 1:
```

Comment:
deprecated since Aug 2022

Reply:
same here. These should be separate changes